web/src/common/colors.ts

```diff
@@ -17,8 +17,10 @@ let commonQuintileColorScale = [
   "#CDE594",
 ];
 
+// The percentiles are [1, 100]. The 5 colors cover 4 each. Notice parts of the
```

Comment:
> The 5 colors cover 4 each.

I know this comment was pre-existing, but I don't understand what it means.

Reply:
I deleted this sentence entirely. I also don't know what I meant anymore; I think I was trying to talk myself through the bucketed legend or something.